TITLE:
wandb.history.add produces confusing error message

LABEL:
c:misc

STATE:
closed

BODY:
Should say module object has no attribute 'history', but we most have history in wandb. Let's fix that.

Traceback (most recent call last):
  File "./run_pg.py", line 79, in <module>
    run_policy_gradient_algorithm(env, agent, callback=callback, usercfg = cfg)
  File "/Users/shawn/code/modular_rl/modular_rl/core.py", line 117, in run_policy_gradient_algorithm
    if callback: callback(stats)
  File "./run_pg.py", line 74, in callback
    wandb.history.add(stats.items())
AttributeError: 'module' object has no attribute 'add'

